core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -57,8 +57,7 @@ class CloudSqlInstance {
   private final ListenableFuture<KeyPair> keyPair;
   private final Object instanceDataGuard = new Object();
   // Limit forced refreshes to 1 every minute.
-  private final RateLimiter<Object> forcedRenewRateLimiter =
-      RateLimiter.burstyBuilder(2, Duration.ofSeconds(30)).build();
+  private final RateLimiter<Object> forcedRenewRateLimiter;
```

Comment:
I added constructor that allows tests to pass in the rate limiter so that we could make the tests run faster.